frontend/src/pages/vpn-relay-welcome.page.tsx

```diff
@@ -62,13 +59,11 @@ const VpnRelayWelcome: NextPage = () => {
               </div>
 
               <p>{l10n.getString("vpn-relay-go-relay-body")}</p>
-              <LinkButton
-                href={"https://www.mozilla.org/en-US/products/vpn/download/"}
-                target="_blank"
-                className={styles["get-addon-button"]}
-              >
-                {l10n.getString("vpn-relay-go-relay-cta")}
-              </LinkButton>
+              <Link href={"/"} target="_blank">
```

Comment:
```suggestion
              <Link href={"/"}>
```

The `target` would have to be on the `<a>`, though I'm not sure if we want it to open in a new tab anyway, since it's on the same site. (But not strong preference.)